mlflow/metrics/metric_definitions.py

```diff
@@ -223,8 +216,13 @@ def _rougeL_eval_fn(eval_df, metrics):
         )
 
 
-def _rougeLsum_eval_fn(eval_df, metrics):
-    if "target" in eval_df:
+def _rougeLsum_eval_fn(predictions, targets, metrics):
+    if targets is not None and len(targets) != 0:
+        if not _validate_text_data(targets, "rougeLsum") or not _validate_text_data(
```

Comment:
Do we need this `if targets is not None and len(targets) != 0:` check?
Can `_validate_text_data(targets, "rougeLsum")` already handle that?

Reply:
Also is there a reason why only some of the metrics have this check to validate if the target has values that are string where other metrics we just check for `if targets is not None and len(targets) != 0:`?

Reply:
Some metrics are solely defined for text, while others are not defined for text, and some are defined for both text and numeric values